src/wallet/asyncrpcoperation_sendmany.h

```diff
@@ -50,7 +50,7 @@ struct WitnessAnchorData {
 
 class AsyncRPCOperation_sendmany : public AsyncRPCOperation {
 public:
-    AsyncRPCOperation_sendmany(std::string fromAddress, std::vector<SendManyRecipient> tOutputs, std::vector<SendManyRecipient> zOutputs, int minDepth, CAmount fee = ASYNC_RPC_OPERATION_DEFAULT_MINERS_FEE);
+    AsyncRPCOperation_sendmany(std::string fromAddress, std::vector<SendManyRecipient> tOutputs, std::vector<SendManyRecipient> zOutputs, int minDepth, CAmount fee = ASYNC_RPC_OPERATION_DEFAULT_MINERS_FEE, Value contextInfo = Value::null);
```

Comment:
Comment (not change request): This API strikes me as odd, because the constructor args should be derived from `contextInfo`, but there's no way to enforce that. This means the caller could accidentally pass the wrong `contextInfo`, and this code will happily relay it.

Here are some alternatives to consider in the future:
- pass `contextInfo` to the ctor then derive the other values from it. (Is this too messy because the ctor would need access to the wallet?)
- derive the context info from the given parameters, so the RPC interface will be returning values that went through a "round trip" of parsing, verification, update. This could avoid the problem I mention above, although sometimes it's nice to know that we're seeing the *original* client request info, just in case that reveals malformed values that are obscured by the "round trip" processing.

For now, let's keep this design since it should work well enough.